src/pages/Map/MapPage.tsx

```diff
@@ -1,26 +1,31 @@
+import Weather from 'api/services/Weather';
+import { useLocalStorage } from 'hooks/useLocalStorage';
 import { TileLayer } from 'react-leaflet';
 import { PlaceMarkers } from './components/PlaceMarkers';
 import { StyledMapContainer, MapWrapper } from './MapPage.styled';
 import { MapPageTestIds } from './MapPageTestIds';
 
-const initialBounds = '(50.72167742756552,16.311950683593754,51.48993452350156,17.76351928710938)';
 const initialZoom = 10;
 
 const MapPage = () => {
-  const localization: [number, number] = [51.107, 17.038];
+  const [localValue] = useLocalStorage('current_location');
```

Comment:
**suggestion** change name of this variable to smth more descriptive like currentLocation

Reply:
**suggestion** to avoid `jumping` map maybe better option will be to store another item in localstorage with coordinates of current location?